### PR TITLE
Fixes broken core docs links - closes #473 

### LINKED ISSUE #473: Fixes broken Core docs link
Fixes broken core docs links

### PR DESCRIPTION
**Pull request by chris529**
Closes #473

### CODE REVIEW

File: modules/ROOT/pages/interact-with-network.adoc [hunk 1]
```diff
@@ -24,6 +25,8 @@ Mona Bärenfänger <mona@lightcurve.io>
 == What does it mean to interact with the network?
 
 Every action that inserts data into the blockchain, or alternatively extracts data from the blockchain is considered an interaction within the blockchain network.
+The "Interact with the network" page explains the different possibilities for other applications to
```
Comment (Tschakki) [inline modules/ROOT/pages/interact-with-network.adoc]: This is a weird line break. Please use only 1 sentence per line :)

File: modules/ROOT/pages/interact-with-network.adoc [hunk 2]
```diff
@@ -37,10 +40,15 @@ Their nodes provide them with a private API that can be used to perform differen
 Depending on their preferences, node operators may wish to use <<javascript,Lisk Elements>>, <<commandline,Lisk Commander>> or a graphical interface such as <<lisk_desktop,Lisk Desktop>>.
 . *Applications* interact with the network through the API.
 For convenience, applications may use wrappers such as xref:{url_elements_pkg_api}[@liskHQ/lisk-api-client].
+A wrapper is a library that holds methods to easily
```
Comment (Tschakki) [inline modules/ROOT/pages/interact-with-network.adoc]: Here is another weird line break

File: modules/ROOT/pages/setup/binary.adoc [hunk 3]
```diff
@@ -38,7 +38,7 @@ include::partial$core-ports.adoc[]
 [source,bash]
 ----
 sudo apt-get update
-sudo apt-get install curl wget tar unzip zip ntp
+brew install curl wget tar unzip zip ntp
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/binary.adoc:49]: this is only macOS specific. Please use the tab-element here to describe both methods:
for Linux with `apt get` and for macOS with `brew`

File: modules/ROOT/pages/setup/binary.adoc [hunk 4]
```diff
@@ -77,18 +77,15 @@ Download Lisk Core:
...
 [source,bash]
 ----
 bash installLisk.sh install -r beta
 ----
 --
-Mainnet::
-+
---
-See xref:{url_core_2_setup}[Lisk Core 2.1.5 binary setup]
---
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/binary.adoc]: Why remove this?

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please see my comments :)

File: modules/ROOT/pages/setup/binary.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/pages/setup/binary.adoc]: After discussions with Michiel, he suggested that we should only have the Betanet here and not the Mainnet.

File: modules/ROOT/pages/setup/binary.adoc (hunk 3 above)
Comment (chris529) [reply modules/ROOT/pages/setup/binary.adoc:49]: Am not sure how to set up the tabs in Webstorm yet, so I have added in 2 separate sentences  which describes which system is required for each command, followed by the corresponding relevant commands. Please have a look and see if this is OK Mona.

Comment (Tschakki) [reply modules/ROOT/pages/setup/binary.adoc:49]: It's ok for now, we can add the tabs later.

File: modules/ROOT/pages/setup/binary.adoc [hunk 5]
```diff
@@ -35,11 +35,19 @@ include::partial$core-ports.adoc[]
 
 === Install dependencies
 
+If you are running Ubuntu then execute the following command:
+
 [source,bash]
 ----
 sudo apt-get update
-sudo apt-get install curl wget tar unzip zip ntp
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/binary.adoc:41]: This line should not be removed

File: modules/ROOT/pages/setup/binary.adoc (hunk 4 above)
Comment (Tschakki) [reply modules/ROOT/pages/setup/binary.adoc]: I do not agree on this. Right now, we already link to the 2.1.5 version here, which is totally fine imho.
Please undo these changes for now, we can discuss it further in the next docs weekly!

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/setup/binary.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/pages/setup/binary.adoc]: OK,  I have done this. However, please check it Mona, as I am not convinced this is 100% correct. many thanks!

File: modules/ROOT/pages/setup/binary.adoc (hunk 5 above)
Comment (chris529) [reply modules/ROOT/pages/setup/binary.adoc:41]: As far as I can see, this command on line 41 it has already been removed

Comment (Tschakki) [reply modules/ROOT/pages/setup/binary.adoc:41]: Yes, as mentioned above, I have now added it back in.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/setup/binary.adoc (hunk 5 above)
Comment (chris529) [reply modules/ROOT/pages/setup/binary.adoc:41]: Ah yes...sorry. OK, it is now added back in :-)

File: modules/ROOT/pages/setup/binary.adoc (hunk 4 above)
Comment (Tschakki) [reply modules/ROOT/pages/setup/binary.adoc]: I pushed a fix to your branch :)

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**